pages/portfolio.html

```diff
@@ -5,8 +5,47 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
```

Comment:
Make sure to change the title for each page. This is the text that appears in the tab in the browser.